### PR TITLE
Disable `SaaS` version of My Account settings by default

### PR DESCRIPTION
**Pull request by brionmario**
### Proposed changes in this pull request

[List all changes you want to add here. If you fixed an issue, please
add a reference to that issue as well.]

- Default config for https://github.com/wso2/identity-apps/pull/6094

### When should this PR be merged

[Please describe any preconditions that need to be addressed before we
can merge this pull request.]

After https://github.com/wso2/identity-apps/pull/6094 is merged.

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 1]
```diff
@@ -1133,7 +1133,10 @@
   ],
   "console.applications.scopes.update": ["internal_application_mgt_update"],
   "console.applications.scopes.delete": ["internal_application_mgt_delete"],
-  "console.applications.disabled_features": ["applications.loginFlow.legacyEditor"],
+  "console.applications.disabled_features": [
+    "applications.loginFlow.legacyEditor",
```
Comment (pavinduLakshan) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:1137]: Shall we add `loginFlow.ai`[1] as well?

[1] https://github.com/wso2/identity-apps/pull/6094/files#diff-e66c115a503827bbd4e1bd6ad04d6ff16ed77f158ae3608fae0f786905382f63L346-R346

**Review by pavinduLakshan — DISMISSED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json (hunk 1 above)
Comment (brionmario) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:1137]: Shall i add it in the same PR?

cc: / @DonOmalVindula

Comment (brionmario) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:1137]: Added `"branding.ai"` as well.

cc:/ @chamathns

Comment (chamathns) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:1137]: +1

**Review by pavinduLakshan — DISMISSED** [review]

**Review by dasuni-30 — APPROVED** [review]

**Review by Yasasr1 — APPROVED** [review]

**Pull request merged**